src/client.ts

```diff
@@ -1,16 +1,17 @@
 import type { GlobalRootLinks, OctopusError, RootResource, SpaceRootLinks, SpaceRootResource } from "@octopusdeploy/message-contracts";
 import ApiClient from "./apiClient";
-import type { RouteArgs } from "./resolver";
-import type { Callback } from "./subscriptionRecord";
 import type { ClientConfiguration } from "./clientConfiguration";
+import { processConfiguration } from "./clientConfiguration";
 import type { ClientErrorResponseDetails } from "./clientErrorResponseDetails";
 import type { ClientRequestDetails } from "./clientRequestDetails";
 import type { ClientResponseDetails } from "./clientResponseDetails";
 import { ClientSession } from "./clientSession";
 import Environment from "./environment";
+import { Logger } from "./logger";
+import type { RouteArgs } from "./resolver";
 import Resolver from "./resolver";
```

Comment:
These should be able to be merged together into a single line and the `type` removed